### PR TITLE
Fix/background image

### PR DESCRIPTION
**Pull request by melohagan**
## Description
I was going to document the background image component, but it didn't appear to work as users' might expect.

Personally I felt that it made more sense for this component to be removed and a container setting to be added instead.

Otherwise, it may still be best to remove the background image component and allow users to create a custom component that can take children.

Addresses: 
- https://github.com/Budibase/budibase/issues/7972

## Screenshots
Screenshot 2023-01-30 at 12 18 01

*In this case, the screen has one background image, and the container has another.*

## Documentation
- [x] I have reviewed the budibase documentatation to verify if this feature requires any changes. If changes or new docs are required I have written them.

*I will update the docs if this change is approved.*

### CODE REVIEW

File: packages/client/manifest.json [hunk 1]
```diff
@@ -998,70 +998,6 @@
       }
     ]
   },
-  "backgroundimage": {
```
Comment (shogunpurple) [inline packages/client/manifest.json:1001]: For backwards compat, we might be better just to keep this around in the manifest but not expose the component in the builder - seems like over 100 are created monthly.

Screenshot 2023-01-30 at 16 21 58

File: packages/client/src/utils/styleable.js [hunk 2]
```diff
@@ -6,7 +6,9 @@ import { builderStore } from "stores"
 const buildStyleString = (styleObject, customStyles) => {
   let str = ""
   Object.entries(styleObject || {}).forEach(([style, value]) => {
-    if (style && value != null) {
+    if (style === "background-image-url" && value != null) {
+      str += `background-image: url("${value}"); background-repeat: no-repeat; background-size: cover; `
```
Comment (shogunpurple) [inline packages/client/src/utils/styleable.js:10]: Hm, this seems like quite a dangerous place to do this - is it not possible to add this directly into the container component to encapsulate a bit?

Comment (melohagan) [reply packages/client/src/utils/styleable.js:10]: I had a go at moving this code into Container.svelte but I wasn't able to get it working. I'm not sure of the order in which the styling is set.

Maybe @aptkingston has an opinion?

File: packages/client/manifest.json (hunk 1 above)
Comment (melohagan) [reply packages/client/manifest.json]: Manifest entry restored.

File: packages/client/src/utils/styleable.js (hunk 2 above)
Comment (aptkingston) [reply packages/client/src/utils/styleable.js:10]: Sorry I was completely sure I had replied to this...

I would prefer not to add in this sort of edge case, but it's reasonably safe if we need to. I would probably rather extend the style definitions to support something like a `format` key, so for URL you could define it as:
```js
    {
      control: Input,
      label: "Image URL",
      key: "background-image",
      format: url => url ? `url("${url}"); background-repeat: no-repeat; background-size: cover;` : ""
    },
```
Which would then require no changes to `styleable.js` and also be reusable in future.

**Comment by melohagan** [comment]
Closing in favour of: https://github.com/Budibase/budibase/pull/9981

**Pull request closed without merging**